dev-python/pychart/pychart-1.39.recipe

```diff
@@ -1,29 +1,63 @@
-DESCRIPTION="PyChart is a Python library for creating high quality charts."
-HOMEPAGE="http://home.gna.org/pychart"
-SOURCE_URI="http://download.gna.org/pychart/PyChart-1.39.tar.gz"
-REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
-CHECKSUM_MD5="f1f509a1c4623056c8e780bb7c9a05c5"
-BUILD()
-{
-	cd PyChart-1.39
-	python setup.py config
-	python setup.py build
+SUMMARY="A Python library for creating high quality charts"
+DESCRIPTION="PyChart is a Python library for creating professional quality PS, PDF, \
+PNG, or SVG charts.  It supports line plots, bar plots, range-fill \
+plots, and pie charts."
+HOMEPAGE="https://github.com/Infinite-Code/PyChart"
+COPYRIGHT="1999-2006 Yasushi Saito"
+LICENSE="GNU GPL v2"
+REVISION="2"
+srcGitRev="8297f66be4777612438c8a3d50940de0eb83f91f"
+SOURCE_URI="https://github.com/Infinite-Code/PyChart/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1fa4470a3afd9fdc09d3280776b61c1bc2bba042dbc97bb6be90e27b2d35d024"
+SOURCE_FILENAME="PyChart-$srcGitRev.tar.gz"
```

Comment:
Since you are editing this recipe, could you try the latest snapshot?
```diff
-srcGitRev="8297f66be4777612438c8a3d50940de0eb83f91f"
+srcGitRev="b4613586ba193a1c8860e1e06c0190246325236b"
 SOURCE_URI="https://github.com/Infinite-Code/PyChart/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="1fa4470a3afd9fdc09d3280776b61c1bc2bba042dbc97bb6be90e27b2d35d024"
+CHECKSUM_SHA256="7c8d7d0b7ca0843b07f1780382a161990b97e998f8b45a8f0ff91c1cf449343c"
-SOURCE_FILENAME="PyChart-$srcGitRev.tar.gz"
+SOURCE_FILENAME="PyChart-$portVersion-$srcGitRev.tar.gz"
 SOURCE_DIR="PyChart-$srcGitRev"
+
 ARCHITECTURES="any"
```

Reply:
And don't forget to bump revision.